src/app/components/landing-page/landing-page.component.html

```diff
@@ -13,10 +13,12 @@
     <div class="flex flex-col items-center gap-4">
       <button
         class="bg-white flex justify-center p-4 items-center text-lg text-coral w-3/4 rounded-full"
+        [routerLink]="['/login']"
```

Comment:
Since the route is a string literal, routerLink does not require evaluation. `routerLink = '/login'`